test/cli_test.py

```diff
@@ -3,10 +3,10 @@
 import shutil
 
 from beets import util
-from beets.mediafile import MediaFile
 from beets.util import bytestring_path, syspath
-from beets.util.confit import ConfigValueError
+from confuse import ConfigValueError
```

Comment:
Could you run `poetry add confuse mediafile` to make them explicit dependencies and add the changes to this PR? Technically it’s not necessary since `beets` already depends on them. But I think it’s good practice to explicitly list all imported packages.